### PR TITLE
Filter pedigree relationship edges such that the unknown terminal node must be an accession

### LINKED ISSUE #1062: On Stock page, cross name shouldn't be included as a descendant in the "Descendants" section

### PR DESCRIPTION
**Pull request by dauglyon**
Description 
-----------------------------------------------------

Fixes issue where crosses and populations would appear in the pedigree tree.

closes #1062 

Checklist 
---------------------------------------------------------------------------------------
- [ ] Documentation only
- [ ] Fixture update only
- [x] Bug fix
  - [x] The relevant issue has been closed.
  - [ ] Further work is required.
- [ ] New feature
  - [ ] Relevant tests have been created and run.
  - [ ] Data was added to the fixture
    - [ ] Data was added via a patch in `/t/data/fixture/patches/`.
  - [ ] User-Facing Change
    - [ ] The user manual in `/docs` has been updated.
  - [ ] Any new Perl has been documented using **perldoc**.
  - [ ] Any new JavaScript has been documented using **JSDoc**.

### CODE REVIEW

File: lib/SGN/Controller/AJAX/Pedigrees.pm [hunk 1]
```diff
@@ -280,11 +280,12 @@ sub get_full_pedigree_GET {
     my $schema = $c->dbic_schema("Bio::Chado::Schema");
     my $mother_cvterm = $schema->resultset("Cv::Cvterm")->find({name  => "female_parent"})->cvterm_id();
     my $father_cvterm = $schema->resultset("Cv::Cvterm")->find({name  => "male_parent"})->cvterm_id();
+    my $accession_cvterm = $schema->resultset("Cv::Cvterm")->find({name  => "accession"})->cvterm_id();
```
Comment (nickmorales) [inline lib/SGN/Controller/AJAX/Pedigrees.pm]: generally we use $accession_cvterm_id = SGN::Model::Cvterm->get_cvterm_row($schema, 'accession', 'stock_property')->cvterm_id(); to fetch cvterms that way you also specify the cv of the cvterm

Comment (titima15) [reply lib/SGN/Controller/AJAX/Pedigrees.pm]: 'accession' is a 'stock_type', not a 'stock_property'

**Comment by dauglyon** [comment]
@nickmorales fixed, sorry I didn't get to this before I left.

File: lib/SGN/Controller/AJAX/Pedigrees.pm [hunk 2]
```diff
@@ -280,11 +281,12 @@ sub get_full_pedigree_GET {
     my $schema = $c->dbic_schema("Bio::Chado::Schema");
     my $mother_cvterm = $schema->resultset("Cv::Cvterm")->find({name  => "female_parent"})->cvterm_id();
```
Comment (nickmorales) [inline lib/SGN/Controller/AJAX/Pedigrees.pm]: sorry but when fetching any cvterm, please use SGN::Model::Cvterm. you can see the available cvterms and cv in system_cvterms.txt

Comment (dauglyon) [reply lib/SGN/Controller/AJAX/Pedigrees.pm]: I'm not sure why I did this think to change all of them, I'll get on that.

**Pull request merged**